dev/com.ibm.ws.jdbc/src/com/ibm/ws/rsadapter/AdapterUtil.java

```diff
@@ -821,11 +821,15 @@ public static SQLException toSQLException(ResourceException resX)
             // The J2C connection manager raises a special exception subclass.
             else if (linkedX instanceof ResourceAllocationException
                      && linkedX.getClass().getName().equals("com.ibm.websphere.ce.j2c.ConnectionWaitTimeoutException")) {
-                // Use a special SQLException subclass for connection wait timeout. 
-                sqlX = new ConnectionWaitTimeoutException(linkedX.getMessage()); 
-                sqlX.initCause(resX); 
-
-                sqlX = new SQLTransientConnectionException(linkedX.getMessage(), "08001", 0, sqlX); 
+                // With the heritageAPIs-1.0 feature in our commercial offering, a change was made from throwing an SQLTransientConnectionException 
+                // to a ConnectionWaitTimeoutException to allow for traditional WebSphere applications to work since they expect a 
+                // ConnectionWaitTimeoutException instead.  This will continue to work with existing liberty applications as well since 
```

Comment:
Lets remove these references to heritageAPIs-1.0 and the commercial offering -- this repo shouldn't make any explicit references to CL or its features